web/src/compose_recipient.ts

```diff
@@ -263,10 +259,8 @@ function get_options_for_recipient_widget(): Option[] {
         name: $t({defaultMessage: "Direct message"}),
     };
 
-    if (
-        realm.realm_private_message_policy ===
-        settings_config.private_message_policy_values.by_anyone.code
-    ) {
+    const {name} = user_groups.get_user_group_from_id(realm.realm_direct_message_permission_group);
+    if (name !== "role:nobody") {
```

Comment:
This check is OK for now but probably not the ideal way to check if nobody can authorize direct messages.

Can you do a follow-up commit that exports a function interface, `user_groups.is_empty_group(realm.realm_direct_message_permission_group)`, replacing all the `role:nobody` checks in the web frontend? 

That implementation should probably work by just walking the actual membership to see if it's empty.